lint-staged.config.js

```diff
@@ -3,7 +3,7 @@ const lintStagedConfig = {
 	"*.{js,json,md,ts,yaml,yml}": "npm run format:write",
 	"*.{js,ts}": ["npm run lint:fix"],
 	// run testing on all supported file types within the src/ directory
-	"src/**/*.{js,ts}": ["npm run test -- --findRelatedTests"],
+	"src/**/*.{js,ts}": ["npm run test -- --findRelatedTests --passWithNoTests"],
```

Comment:
This should be only for this PR since we migrate to Typescript action, right? I'd expect us not to pass without any tests.

Reply:
Removed this in the [last commit](https://github.com/1Password/install-cli-action/pull/16/commits/e06540d5a7bf1439e1f010d8896f2f1120d43e8b). Tests will be required.